notifications-ucr.html

```diff
@@ -674,6 +753,12 @@ <h3 property="schema:name skos:prefLabel">Functional Requirements</h3>
                     <dd about="#functional-requirement-authorization" property="skos:definition">Authorization is enforced on all subscriptions.</dd>
                     <dt about="#functional-requirement-capability-discovery" property="skos:definition" typeof="skos:Concept">F6. <dfn id="functional-requirement-capability-discovery">Capability Discovery</dfn></dt>
                     <dd about="#functional-requirement-capability-discovery" property="skos:definition">A client is able to discover the notification capabilities of a Solid Storage.</dd>
+                    <dt about="#functional-requirement-request-handling" property="skos:definition" typeof="skos:Concept">F7.
+                    <dfn id="functional-requirement-request-handling">Request Handling</dfn></dt>
+                    <dd about="#functional-requirement-request-handling" property="skos:definition">The Solid storage must be able to process the parameters of the subscription request made by a client.</dd>
+                    <dt about="#functional-requirement-request-handling" property="skos:definition" typeof="skos:Concept">F8.
+                    <dfn id="functional-requirement-request-modification-handling">Request Modification Handling</dfn></dt>
+                    <dd about="#functional-requirement-request-modification-handling" property="skos:definition">The Solid storage must be able to modify the parameters of the subscription request made by a client.</dd>
```

Comment:
If the client request can be modified by a server, the the client request is no more than a hint. Is that the intention?

What would conformance look like if client requests are treated as hints?

Reply:
That is not the intention. The idea is that after a subscription has been established, the client can subsequently request the server to modify the features (and maybe list of resource, if we choose to support that) of the said subscription. This requires that the server must have the ability to process these subsequent requests and modify the notifications accordingly. One example could be that after a period of user inactivity, the client could ask the server to reduce the rate of notifications to something like once in 5 minutes (instead of, say, an original of every second). 

We might also decide to solve this problem by asking the client to establish a new connection instead. But that is a design decision and needs to be documented in the standard itself.

We could definitely do with better terminology + language here...

Reply:
See new commits for improved language...